x-pack/plugins/security_solution/cypress/support/es_archiver.ts

```diff
@@ -0,0 +1,59 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import { EsArchiver } from '@kbn/es-archiver';
+import { KbnClient } from '@kbn/test';
+import { Client, HttpConnection } from '@elastic/elasticsearch';
+import { ToolingLog } from '@kbn/tooling-log';
+
+export const esArchiver = (
+  on: Cypress.PluginEvents,
+  config: Cypress.PluginConfigOptions
+): EsArchiver => {
+  const log = new ToolingLog({ level: 'verbose', writeTo: process.stdout });
+
+  const client = new Client({
+    node: config.env.ELASTICSEARCH_URL,
+    Connection: HttpConnection,
+  });
+
+  const kbnClient = new KbnClient({
+    log,
+    url: config.env.CYPRESS_BASE_URL as string,
+  });
+
+  const esArchiverInstance = new EsArchiver({
+    log,
+    client,
+    kbnClient,
+    baseDir: '../../../test/security_solution_cypress/es_archives',
```

Comment:
What do you think about not hardcoding this value here and instead putting it on the cypress config json and read it from there?

Reply:
🤷 it works the same way as before, if there is a case for that you can always extend in the future